zebra-network/src/config.rs

```diff
@@ -37,19 +32,53 @@ pub struct Config {
 }
 
 impl Config {
-    fn parse_peers<S: ToSocketAddrs>(peers: HashSet<S>) -> HashSet<SocketAddr> {
-        peers
-            .iter()
-            .flat_map(|s| s.to_socket_addrs())
-            .flatten()
-            .collect()
+    async fn parse_peers(peers: HashSet<String>) -> HashSet<SocketAddr> {
```

Comment:
Let's document this function:
```suggestion
    /// Resolves `host` into zero or more IP addresses.
    ///
    /// If `host` is a DNS name, performs DNS resolution with a timeout of a few
    /// seconds. If DNS resolution fails or times out, returns an empty list.
    async fn resolve_peers(peers: HashSet<String>) -> HashSet<SocketAddr> {
```